spec/models/zone_spec.rb

```diff
@@ -192,6 +192,28 @@
     end
   end
 
+  context "maintenance zone" do
+    it "creates missing region during seed" do
+      expect(MiqRegion.my_region).to be_nil
+      described_class.seed
+      expect(MiqRegion.my_region).to be_present
+    end
+
+    it "is seeded with relation to region" do
+      described_class.seed
+      zone = Zone.where("name LIKE ?", "#{Zone::MAINTENANCE_ZONE_NAME_PREFIX}%").first
```

Comment:
I still don't think this should be in a constant

Reply:
well, magic constants were forbidden even in school and I'm not a fan of them too :)

Reply:
https://getyarn.io/yarn-clip/1b7d74e5-04f6-469b-93fc-be299e892929/gif

Reply:
I disagree with you, but this GIF convinced me :laughing:

Reply:
What makes this a "magic constant"?  It's just a frozen string.  It's always the same frozen string.